code-challenges.js

```diff
@@ -7,43 +7,130 @@
+// Then console.log the new variable addNumbers along with the length operator .length in parenthesis with a semicolon outside to tell the expected function arguements in myNumbers1.
+// Then console.log myNumbers1 to count the elements inside the newly combined array.
+// Therefore these codes meet the requirements to output the given outcome 10.
+
+
+// var addNumbers = myNumbers1.concat(myNumbers2)
+// console.log(addNumbers.length);
+// console.log(myNumbers1)
 
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 ahplA"
 
-const currentCohort = "Alpha 2022"
+// const currentCohort = "Alpha 2022"
+
+// Pseudocode:
+// Create a code that will reverse the letters of a string.
+// Make a variable named alpha equating to the currentCohort to turn a string into an array using .split("") Acessor with quotation mark in the parethesis.
+// Then console.log using .reverse() to make the array order go in reverse using the new variable alpha prior the mutator indicating as Alpha 2022.
+// Then conmsole.log using .join(" ") with a space to separate the characters and make a string by converting the values.
+//Therefore the following codes meet the conditions to output 2202 ahplA.
+
+
+// var alpha= currentCohort.split("")
+// console.log(alpha.reverse())
+// console.log(alpha.join(" "))
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-const myArray = [13, 34, 5, 10, 27, 42]
+// const myArray = [13, 34, 5, 10, 27, 42]
+
+// Pseudocode:
+// Create a statement that evaluates whether it's even or odd within the given numbers of the array.
+// Use a for loop declaration stating that arrays myArray is less than one to multiply.
+// Use the declaration const name num that equates to myArray with a value of i = 0 in a bracket to state that identifier doesn't change.
+// Use an If/Else statement to see if the program will continue to run when the set condition is true, plugging in the new const name nuum modulo 2 with a Loose Equality operatorn == and 0 all being in parenthese and this will compare the two values for equality.
+// Then console.log with parentheses sqying the number is even in quotation for even numbers.
+// Then console.log with parentheses saying the number is odd in quotation for odd numbers.
+// Therefore the output desired is displaying in the terminal because the conditions are met.
+
+
+// for(let i=0; i<myArray.length; i++){
+// const num = myArray[i]
+// if(num % 2 == 0) {
+//     console.log("even")
+// }
+// else
+//     console.log("odd")
+// }
```

Comment:
Be aware of your indentation. Everything inside a code block should be indented.

Reply:
oh i see, i will keep that in my mind.